src/logic/policies/data.ts

```diff
@@ -124,27 +124,30 @@ export function useUserPolicyById(policyId: string) {
 
 async function loadPolicies(contract: ClaimsManager, params: { userAccount?: string; policyId?: string }) {
   const { userAccount = null, policyId = null } = params;
-  const [createdEvents, upgradedEvents] = await Promise.all([
+  const [createdEvents, upgradedEvents, downgradedEvents] = await Promise.all([
     contract.queryFilter(contract.filters.CreatedPolicy(null, userAccount, policyId)),
     contract.queryFilter(contract.filters.UpgradedPolicy(null, userAccount, policyId)),
+    contract.queryFilter(contract.filters.DowngradedPolicy(null, userAccount, policyId)),
   ]);
 
   if (!createdEvents.length) {
     return { ids: [], byId: {} };
   }
 
+  const stateChangedEvents = sortEvents([...upgradedEvents, ...downgradedEvents] as const);
```

Comment:
Nit: Do you need as const here?

Reply:
Yeah this one was unexpected. Without `as const` TS incorrectly infers `stateChangedEvents` as `UpgradedPolicyEvent[]`. With `as const` TS correctly infers it as `UpgradedPolicyEvent | DowngradedPolicyEvent)[]`.

Reply:
Oh, I see. That seems one of the many bugs they have.